### ISSUE TITLE
Centralsettlement-service is failing GP tests when enabling event-sdk sidecar on v13.0.1 release candidate

### ISSUE THREAD

**Issue by mdebarros** [mojaloop/project#2368]
**Summary**:

Centralsettlement-service is failing GP tests when enabling event-sdk sidecar on v13.0.1 release candidate.

```bash
Test Suite:[33mGP Tests[39m
Environment:[33mDevelopment[39m
┌───────────────────────────────────────────────────┐
│                      SUMMARY                      │
├───────────────────┬───────────────────────────────┤
│ Total assertions  │ 2064                          │
├───────────────────┼───────────────────────────────┤
│ Passed assertions │ 1926                          │
├───────────────────┼───────────────────────────────┤
│ Failed assertions │ 138                           │
├───────────────────┼───────────────────────────────┤
│ Total requests    │ 446                           │
├───────────────────┼───────────────────────────────┤
│ Total test cases  │ 99                            │
├───────────────────┼───────────────────────────────┤
│ Passed percentage │ 93.31%                        │
├───────────────────┼───────────────────────────────┤
│ Started time      │ Tue, 03 Aug 2021 15:26:55 GMT │
├───────────────────┼───────────────────────────────┤
│ Completed time    │ Tue, 03 Aug 2021 15:29:16 GMT │
├───────────────────┼───────────────────────────────┤
│ Runtime duration  │ 141306 ms                     │
└───────────────────┴───────────────────────────────┘
```

Workaround: disable sidecar (which is already the case by default).

**Severity**:
Low

**Priority**:
Medium

**Expected Behavior**

GP tests should pass when the sidecar has been enabled on the Centralsettlement-service.

**Steps to Reproduce**
  1. Deploy release/v13.0.1 branch, ensuring that Centralsettlement-service.sidecar.enable=true in the associated values.yaml file
  2. Run `helm test` command

**Specifications**
  - Component (if known): Central-Settlements
  - Version: mojaloop/central-settlement:v13.4.0
  - Platform: n/a
  - Subsystem: Helm v13.0.1 candidate release
  - Type of testing: Manual
  - Bug found/raised by: @mdebarros 

**Notes**:

- Severity when opened: Low
- Priority when opened: Medium

**Comment by mdebarros** [comment]
PR to fix issue: https://github.com/mojaloop/central-settlement/pull/368

**Comment by mdebarros** [comment]
> PR to fix issue: [mojaloop/central-settlement#368](https://github.com/mojaloop/central-settlement/pull/368)

GP now passes in PR -> https://github.com/mojaloop/helm/pull/445

```text
[33m--------------------FINAL REPORT--------------------[39m

Test Suite:[33mGP Tests[39m
Environment:[33mDevelopment[39m
┌───────────────────────────────────────────────────┐
│                      SUMMARY                      │
├───────────────────┬───────────────────────────────┤
│ Total assertions  │ 2064                          │
├───────────────────┼───────────────────────────────┤
│ Passed assertions │ 2064                          │
├───────────────────┼───────────────────────────────┤
│ Failed assertions │ 0                             │
├───────────────────┼───────────────────────────────┤
│ Total requests    │ 446                           │
├───────────────────┼───────────────────────────────┤
│ Total test cases  │ 99                            │
├───────────────────┼───────────────────────────────┤
│ Passed percentage │ 100.00%                       │
├───────────────────┼───────────────────────────────┤
│ Started time      │ Wed, 04 Aug 2021 16:16:04 GMT │
├───────────────────┼───────────────────────────────┤
│ Completed time    │ Wed, 04 Aug 2021 16:18:18 GMT │
├───────────────────┼───────────────────────────────┤
│ Runtime duration  │ 133760 ms                     │
└───────────────────┴───────────────────────────────┘
```
